tfhe/Cargo.toml

```diff
@@ -120,6 +120,9 @@ generator_aarch64_aes = ["concrete-csprng/generator_aarch64_aes"]
 
 # Private features
 __profiling = []
+__coverage = []
+
+coverage = []
```

Comment:
please remove the public feature and keep the private one, switch all usage of the public feature to the private one